mission_control/doc/Groot/README.md

```diff
@@ -0,0 +1,54 @@
+# Groot
+
+This document explains how to install and use [Groot](https://github.com/BehaviorTree/Groot).
+
+## Features
+Groot is a Graphical Editor, written in C++ and Qt, to build and inspect behavior trees.
+It is compliant with the the [BehaviorTree.CPP](https://github.com/BehaviorTree/BehaviorTree.CPP) library. 
+
+The next behaviors/actions were implemented as tree node model (Groot Palette)
+- Abort
+- Attitude Servo
+- Fix Rudder
+- Go To Waypoint
+- Payload Command
+- Set Altitude Heading
+- Set Depth Heading
```

Comment:
@patapufate this is already in the README, no?

Reply:
Fixed